perllib/FixMyStreet/App.pm

```diff
@@ -348,9 +347,19 @@ sub send_email {
 }
 
 sub send_email_cron {
-    my ( $c, $params, $env_from, $env_to, $nomail, $cobrand, $lang_code ) = @_;
-
-    return 1 if $c->is_abuser( $env_to );
+    my ( $c, $params, $env_from, $nomail, $cobrand, $lang_code ) = @_;
+
+    my $first_to;
+    if (ref($params->{To}) eq 'ARRAY') {
+        if (ref($params->{To}[0]) eq 'ARRAY') {
+            $first_to = $params->{To}[0][0];
+        } else {
+            $first_to = $params->{To}[0];
+        }
+    } else {
+        $first_to = $params->{To};
+    }
+    return 1 if $c->is_abuser($first_to);
```

Comment:
Yes, and previously I don't think this was working, because `$env_to` was an arrayref, and it looks like `is_abuser` assumes it's a string. So this is just parsing out a To address to check (and To line can take a few forms annoyingly, and didn't seem worth parsing into an Email object just for that).